package.json

```diff
@@ -81,6 +83,7 @@
     "karma-webpack": "=2.0.3",
     "mocha": "=3.2.0",
     "postcss-loader": "=2.0.6",
+    "postcss-nested": "=2.0.2",
```

Comment:
I tried to remove this plugin and it broke my styles for sign/verify message dialog.
![broken-css](https://user-images.githubusercontent.com/1254342/28105570-4694dc86-66e0-11e7-8bdd-d9e603c66322.png)